docs/models.rst

```diff
@@ -76,14 +72,14 @@ function.  However the foreign key has to be separately declared with the
 
     class Person(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        name = db.Column(db.String(50))
-        addresses = db.relationship('Address', backref='person',
-                                    lazy='dynamic')
+        name = db.Column(db.String(50), nullable=False)
+        addresses = db.relationship('Address', backref='person', lazy=True)
```

Comment:
Just checked, all `backref()` ones are explicit, for simple `backref='name'` cases there's no way to be more explicit without switching to `backref()`